modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -44,6 +44,9 @@ package object dao extends MoreTupleOps with ToUserProgramRoleOps {
   implicit val InstantMeta: Meta[Instant] =
     Meta[Timestamp].nxmap(_.toInstant, Timestamp.from)
 
+  implicit val DurationMeta: Meta[Duration] =
```

Comment:
I wonder if we want to be using `scala.concurrent.duration.FiniteDuration` here, just to avoid JDK classes in the model since they may not work on Scala-JS. I don't have a strong opinion, just wondering if you had considered it.